lib/events/filesessions/fileasync.go

```diff
@@ -500,7 +505,7 @@ func (u *Uploader) upload(ctx context.Context, up *upload) error {
 			if !trace.IsNotFound(err) {
 				return trace.Wrap(err)
 			}
-			u.log.WithError(err).Warningf(
+			log.WithError(err).Warningf(
```

Comment:
You can probably remove the session ID from the log message since it's now logged as a field.